FrontEnd/Brewerweb/js/CheckFields.js

```diff
@@ -0,0 +1,127 @@
+/* Justin Evans
+*/
+// This script validates a form.
+
+// Function called when the form is submitted.
+// Function validates the form data.
+function validateForm(e) {
+    'use strict';
+
+    // Get the event object:
+	if (typeof e == 'undefined') e = window.event;
+
+    // Get form references:
+	var firstName = U.$('firstName');
+	var lastName = U.$('lastName');
+	var email = U.$('email');
+	var phone = U.$('phone');
+	var city = U.$('city');
+	var state = U.$('state');
+	var zip = U.$('zip');
+	var terms = U.$('terms');
+
+	// Flag variable:
+	var error = false;
+
+	// Validate the first name:
+	if (/^[A-Z \.\-']{2,20}$/i.test(firstName.value)) {
+		removeErrorMessage('firstName');
+	} else {
+		addErrorMessage('firstName', 'Please enter your first name.');
+		error = true;
+	}
+	//validate the last name bullet 13
+	if (/^[A-Z \.\-']{2,20}$/i.test(lastName.value)) {
+		removeErrorMessage('lastName');
+	} else {
+		addErrorMessage('lastName', 'Please enter your last name.');
+		error = true;
+	}
+	// Validate the email address:
+	if (/^[\w.-]+@[\w.-]+\.[A-Za-z]{2,6}$/.test(email.value)) {

```

Comment:
A user's email will probably not start with www. so this might break everyone's email fields